sdk/include/crt/mingw32/intrin_x86.h

```diff
@@ -833,7 +833,7 @@ __INTRIN_INLINE void __movsd(unsigned long * Destination, const unsigned long *
 {
 	__asm__ __volatile__
 	(
-		"rep; movsd" :
+		"rep; movsl" :
```

Comment:
https://lore.kernel.org/lkml/23d1f8db268d43afa332517779b0ef95@AcuMS.aculab.com/T/

Change is legit.

Reply:
> https://lore.kernel.org/lkml/23d1f8db268d43afa332517779b0ef95@AcuMS.aculab.com/T/

Thanks. Note 'The semicolon after rep isn't needed anymore either ;)'.